tmt/convert.py

```diff
@@ -67,10 +67,10 @@ def read_manual(
 
     try:
         if plan_id:
-            all_cases = nitrate.TestPlan(int(plan_id)).testcases
+            all_cases = nitrate.TestPlan(plan_id).testcases
```

Comment:
Please add type to the click options. Otherwise runtime propagates 'str' here which is not accepted.

Reply:
+1, I was just looking into why it does not work :) thanks

Reply:
added type to the parameters

Reply:
waiting for tests

Reply:
all good \o/